tools/documentation-code-snippets/README.md

```diff
@@ -0,0 +1,3 @@
+# documentation-code-snippets
+
+Updates code snippets in our documentation with pieces from our examples. Mostly using the example called "documentation".
```

Comment:
I wonder if we can make the overall intent clearer here. Like: "We want all of the code in our documentation translated and tested, this is why we have the examples/lang/documentation. To make sure the examples in the docs stays in sync with these examples we created this tool that keeps them in sync"